src/main/User.java

```diff
@@ -0,0 +1,74 @@
+package main;
+
+
+public class User {
+	private String name;
+	private int age;
+	private String email;
+	
+	public static void main(String[] args) {
```

Comment:
Always be mindful of your codebase and try to not leave unnecessary traces of code.